domain/bakerystorage/state/state.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import (
+	"github.com/juju/juju/core/database"
+)
+
+type State struct {
+	*ConfigState
+}
+
+func NewState(factory database.TxnRunnerFactory) *State {
```

Comment:
Why not just put everything in state.go?

Reply:
Same as above, more is coming